packages/yasqe/src/autocompleters/index.ts

```diff
@@ -316,24 +315,28 @@ export const fetchFromLov = (
   //   .append($("<span>Fetchting autocompletions &nbsp;</span>"))
   //   .append($(yutils.svg.getElement(require("../imgs.js").loader)).addClass("notificationLoader"));
   // doRequests();
-  return superagent
-    .get(reqProtocol + "lov.linkeddata.es/dataset/lov/api/v2/autocomplete/terms")
-    .query({
-      q: token.autocompletionString,
-      page_size: 50,
-      type: type,
+
+  const params = new URLSearchParams();
+  params.append("q", token.autocompletionString || "");
```

Comment:
Theoretically, the fallback should be never hit, but let's keep it 👍

Reply:
It is to appease the typescript gods as `token.autocompletionString` can be `undefined`  :) 

Another solution would be to use:

```ts
if (token.autocompletionString) params.append("q", token.autocompletionString);
```